src/js/ctlevent.js

```diff
@@ -22,13 +22,27 @@ var propertiesString = function(properties) {
     return propString;
 };
 
+let compareDates = (start, end) => {
+    let startStr = new Date(start.toLocaleDateString());
+    let endStr = new Date (end.toLocaleDateString());
+
+    if(endStr > startStr){
```

Comment:
we use a convention for if statements... should be like: `if (endStr > startStr) {`

Reply:
On second glance, this whole if statement could be simplified, just replace it with `return endStr > startStr;` to return that boolean.

Reply:
Ha...so right. Thank you, I'll put in the change.